spec/app/domain/authentication/authn-oidc/v2/data_objects/authenticator_spec.rb

```diff
@@ -37,12 +34,12 @@
 
     context 'when initialized with an array argument' do
       context 'single value array' do
-        let(:args) { default_args.merge({ provider_scope: ['foo'] }) }
+        let(:args) { default_args.merge({ provider_scope: 'foo' }) }
```

Comment:
Does it make sense to add a test where provider_scope is set to something that includes 'openid' or 'email' or 'profile' to make sure that duplicates continue to be removed?

Also this test looks identical to the one in line 25, except for the context strings. Is it actually using a single value array?

Reply:
I've added a [test case for duplicated scopes](https://github.com/cyberark/conjur/pull/2662/commits/b0955cc41a4f1577414acac95ba8cfa1f6761d91#diff-4834da11818ee5df3c5345470528c71b8cc9e4189a669ad40ef2256a076913d8R35) in the latest commit.